tsconfig.json

```diff
@@ -37,11 +37,13 @@
   "exclude": [
     "dist",
     "packages/neo-one-server-plugin-wallet/src/contracts/*.ts",
-    "packages/neo-one-smart-contract-lib/src/*.ts",
     "packages/neo-one-smart-contract/src/*.d.ts",
+    "packages/neo-one-smart-contract-lib/src/index.ts",
```

Comment:
Why do we need the `"packages/neo-one-smart-contract-lib/src/*.ts",` change?

Reply:
@dicarlo2  The Test......test.ts files don't contain smart contract logic, tsc errors aren't thrown so i figure this is as good an opportunity as any to un-exclude these file.